index.js

```diff
@@ -23,49 +24,61 @@ const featuredeploy = (args, callback) => {
 }
 
 app.post('/pull_request', (req, res) => {
-  const {action, pull_request, label, repo, installation} = req.body
-  switch (action) {
-    case 'labeled':
-      if (label.name === 'featuredeploy') {
-        integrationTools.makeGithubFeatureDeployComments({
-          installationId: installation.id,
-          pullUrl: pull_request.url,
-          message: 'building...'
-        })
-        featuredeploy(['deploy', pull_request.head.ref, pull_request.head.sha], () => false)
-      }
-      break
-    case 'unlabeled':
-      if (label.name === 'featuredeploy') {
-        integrationTools.removeGithubFeatureDeployComments({
-          installationId: installation.id,
-          pullUrl: pull_request.url
-        })
-        featuredeploy(['rmbranch', pull_request.head.ref], () => false)
-      }
-      break
+  if (req.query.secret === SECRET){
```

Comment:
I don't think we can get github to send this in the request, though  - or is there something we can do?